nodes/liveplot

```diff
@@ -0,0 +1,159 @@
+    '''
+    def __init__(self, ax, buffer_length):
+        self.data = DataHandler(buffer_length)
+        self.lines = {}
+        self.pts = {}
+        self.ax = ax
+ 
+    def _delete_dead_lines(self, objIds_in_data):
+        '''
+        if a objId is not any more in the data obj, we assume that it hade timed out
+        and we delete it here too
+        '''
+        for objId in self.lines.keys():
+            if objId not in objIds_in_data:
+                ind_lines = self.ax.lines.index(self.lines[objId])
+                del self.ax.lines[ind_lines]
+                ind_pts = self.ax.lines.index(self.pts[objId])
+                del self.ax.lines[ind_pts]
+                del self.lines[objId]
+                del self.pts[objId]
+#                print 'we deleted:', objId
+    def draw(self,i):
+        '''
+        lets draw the graph
+        '''
+        self.data.aquire()
+        da = self.data.get_data_to_draw()
+        objIds_in_data = da.keys()   
+        for objId in objIds_in_data:
+            if objId not in self.lines.keys():
+                # add new line
+                self.lines[objId] , = self.ax.plot(da[objId]['x'], da[objId]['y'],da[objId]['z'])
+                self.pts[objId] , = self.ax.plot([da[objId]['x'][-1]], [da[objId]['y'][-1]],[da[objId]['z'][-1]],'o', c="r")
+            # load data from mem to lines
+            self.lines[objId].set_data(da[objId]['x'], da[objId]['y'])
+            self.lines[objId].set_3d_properties(da[objId]['z']) # this f*** line is needet to draw in this plot
+            self.pts[objId].set_data([da[objId]['x'][-1]], [da[objId]['y'][-1]])
+            self.pts[objId].set_3d_properties([da[objId]['z'][-1]]) # this f*** line is needet to draw in this plot
+        self._delete_dead_lines(objIds_in_data)
+        return self.lines, self.pts
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Alternative to rviz, listens to flydra_mainbrain_super_packets')
+    parser.add_argument('-x_min', action='store', dest='x_min', type=int,default=-0.2, help='drawing limits')
+    parser.add_argument('-x_max', action='store', dest='x_max', type=int,default=0.2, help='drawing limits')
+    parser.add_argument('-y_min', action='store', dest='y_min', type=int,default=-0.2, help='drawing limits')
+    parser.add_argument('-y_max', action='store', dest='y_max', type=int,default=0.2, help='drawing limits')
+    parser.add_argument('-z_min', action='store', dest='z_min', type=int,default=-0.2, help='drawing limits')
+    parser.add_argument('-z_max', action='store', dest='z_max', type=int,default=0.2, help='drawing limits')
+    parser.add_argument('-update_rate', action='store', dest='update_rate', type=int,default=10, help='in Hz, should not be higher than 20Hz')
+    parser.add_argument('-buffer_length', action='store', dest='buffer_length', type=int,default=500, help='how many points of a trajectory should be drawn')
+    args = parser.parse_args()
+
```

Comment:
Can you support auto scaling? Also it would be good to fall back to defaults based on ros arguments if the command line arguments are not provided